src/data/chipLists.ts

```diff
@@ -0,0 +1,24 @@
+const chipLists = [
```

Comment:
Is there a specific reason of making this file a ts one not js?

also the other file in the same dir